lib/Onlineorder.rb

```diff
@@ -0,0 +1,90 @@
+require "csv"
+require "ap"
+require_relative "order.rb"
+require_relative "customer.rb"
+
+module Grocery
+  class OnlineOrder < Order
+    attr_accessor :customer, :online_order_id, :products, :fulfillment_status
```

Comment:
A couple of things:
1.  This doesn't need to be an accessor, a reader would be better.
1.  You're already inheriting `products` from `Order` so you don't need that here.
1.  You don't need an `online_order_id` since this **is** an `Order` and thus already has an `id` field.